id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/manager/CredentialServiceManager.java

```diff
@@ -123,6 +123,9 @@ public class CredentialServiceManager {
 	@Value("${" + UIN_REFID + "}")
 	private String uinRefId;
 
+	@Value("${mosip.idrepo.uin.disable-support:false}")
+	private boolean uinSupportDisabled;
```

Comment:
Can we rename this to mosip.idrepo.identity.disable-uin-based-credential-request ? Asking this change because the entire identity service revolves around UIN.

Reply:
ok

Reply:
Change the variable name to "disableUINBasedCredentialRequest"